ext_localconf.php

```diff
@@ -24,3 +24,8 @@
     );
 
 }
+
+$GLOBALS['TYPO3_CONF_VARS']['SYS']['fal']['processors']['tinyimg'] = [
+    'className' => \Schmitzal\Tinyimg\Service\LocalImageProcessor::class,
+    'before' => ['LocalImageProcessor']
+];
```

Comment:
i think this should not be done in EXT:tinyimg but on the site-extension of the project (if you want to compress proccessed files)

Reply:
sounds like an option to enable this on ExtensionConfiguration level?